GomokuClient/packages/gomoku-core/src/pages/JoinGame.tsx

```diff
@@ -12,25 +12,35 @@ import * as signalR from "@microsoft/signalr";
 import { useSignalRReconnection } from "@/hooks/useSignalRReconnection";
 
 const JoinGame = () => {
-  const { board, winner, handlePieceClick } = useBoard();
+  const { board, winner, addPiece } = useBoard();
   // TODO: check for a better way to get gameID from url, should be considered routing file params loaders
   const { gameID } = useParams({ strict: false });
   const { jwtToken } = useAuthToken();
   const joinGame = useJoinGame(
-    jwtToken || typedStorage.getItem("jwtToken") || "",
+    jwtToken || typedStorage.getItem("jwtToken") || ""
   );
 
   const { connection, isConnected } = useSignalRConnection();
 
+  useEffect(() => {
+    if (connection && isConnected && gameID) {
+      void connection.invoke("JoinGameGroup", gameID);
```

Comment:
There is no need to use void before a simple execution. Why?
- First, for consistency, we would need to use void everywhere, which would make the code unnecessarily verbose.
- Second, it’s not required here, as TypeScript can infer the type on its own

Reply:
TypeScript shows warning, that returned promise is unhanded with that.

![image](https://github.com/user-attachments/assets/2254974a-e587-4337-8bf7-6ce84b0e4bbc)

Reply:
It's your visual code issue, typechecks CLI should pass, if you remove void here

<img width="707" alt="Screenshot 2024-10-01 at 23 25 56" src="https://github.com/user-attachments/assets/22d53d23-ed95-4c17-bec4-37ec97c7a48b">

Reply:
I use webstorm as well. You see, that invoke line is unrelined? Hover it and it will show up that warning for you as well

Reply:
Well, CLI checks will pass, it's not typescript issue and void should not be here, if it would be typescript issue the CLI will not pass then. Second IDEA suggests to me .then()

<img width="473" alt="Screenshot 2024-10-01 at 23 49 37" src="https://github.com/user-attachments/assets/67e67efd-0341-4787-b746-580686bfad05">

So by placing await will solve that issue

<img width="606" alt="Screenshot 2024-10-01 at 23 48 18" src="https://github.com/user-attachments/assets/c17afc98-f76d-4a13-81fa-ad95d7d530de">

But, it's a case where we don't need to await actually so both options are valid, because we don't need to work with resolved promise here

Reply:
We can have a separate call to discuss if we want  to align IDE behavior with more strict TypeScript or ESLint settings, or to disregard IDE suggestions

Reply:
Remove void for now. Let's ignore and align later